web/js/map/layerbuilder.js

```diff
@@ -387,7 +387,7 @@ export default function mapLayerBuilder(config, cache, store) {
       sourceOptions.tileClass = lookupFactory(lookup, sourceOptions);
     }
     return new OlLayerTile({
-      preload: Infinity,
+      preload: 0,
```

Comment:
Is it necessary to disable this?

Reply:
I should have elaborated on this change in the description to provide some more understanding for why I chose to disable this:

It's not absolutely necessary but, personally I didn't think the slightly improved user experience justified the performance cost if we are going to preload tiles like this.  That said, I think the change warrants a broader discussion among the team.  For reference, I did a quick comparison of resources transferred with `preload` set to both `0` and `Infinity` on this branch to get an idea of the cost in bandwidth between the two possible approaches.  This URL has one baselayer and two vector fire layers turned on.

Using these starting params: `/?v=-128.6060460332096,35.31341649269199,-114.574428132895,47.54658937507575&l=MODIS_Terra_Thermal_Anomalies_Night,MODIS_Terra_Thermal_Anomalies_Day,Reference_Labels_15m,Reference_Features_15m,MODIS_Terra_CorrectedReflectance_TrueColor&lg=false&t=2021-08-08-T02%3A00%3A00Z`

* Initial page load
	* `preload: 0` =  3.0MB
	* `preload: Infinity` = 6.0MB

* 5 day steps
	* `preload: 0` = 11.8MB
	* `preload: Infinity` = 20.0MB

The difference seemed to be about a factor of two.  Considering that, when stepping forward/backwards in time and not modifying the zoom level the app is requesting a lot of tiles that the user might never see, it didn't seem worth it to request almost double the number of tiles.  Especially since the only downside to setting `preload: 0` is that the user will see some blank tiles around the edges when zooming out or, slightly blurry tiles momentarily while zooming in.

But again, I think it's worth more discussion to arrive at a team consensus on this change.